server/utils/user-stats.test.js

```diff
@@ -419,8 +419,8 @@ test('Longest streak calculation', function(t) {
         1454519128123, moment.utc().valueOf()
       ])
     ),
-    4,
-    'should return 4 when there is a break in UTC (but no break in PST)'
+    16,
+    'should return 16 when there is a break in UTC (but no break in PST)'
```

Comment:
This number jumped up because it is now counting for all the times that were previously missed